.github/workflows/ci.yml

```diff
@@ -26,9 +26,9 @@ jobs:
       - name: test
         run: |
           sudo apt-get install tcl8.6 tclx
-          ./runtest --verbose --tags -slow --dump-logs --other-server-path tests/tmp/valkey-7.2.7-noble-x86_64/bin/valkey-server
+          ./runtest --verbose --tags "-slow needs:other-server" --dump-logs --other-server-path tests/tmp/valkey-7.2.7-noble-x86_64/bin/valkey-server
```

Comment:
This is `test-ubuntu-latest`. I guess it shoukd run not only the cross-compat tests...

Reply:
Ah my bad. I think I misread it to be following similar approach to daily.yml i.e. test cross server compatibility separately.